frontend/src/stores/search.ts

```diff
@@ -62,36 +63,28 @@ export interface SearchState {
 /**
  * Builds the search query parameters for the given search type, filters, and search term.
  * `q` parameter is always included as the first query parameter.
+ * If the search type is not supported, only the `q` parameter is included.
  * Only the filters that are relevant for the search type and have a value are included.
  *
- * Some parameters are included in the query depending on the mode:
- * - `INCLUDE_SENSITIVE_QUERY_PARAM` is added to the API search query if the setting is `on`
- * in the featureFlagStore.
+ * `INCLUDE_SENSITIVE_QUERY_PARAM` is never added to the frontend search query. It is added
+ * to the API search query if the setting is `on` in the featureFlagStore.
  */
-function computeQueryParams(
+export function computeQueryParams(
   searchType: SearchType,
   filters: Filters,
   searchTerm: string,
   mode: "frontend" | "API"
 ) {
-  // The filters object is converted to a Record<string, string> object.
-  // e.g., { licenseTypes: [{ code: "commercial", checked: true }] }
-  // => { license_type: "commercial" }
-  const query = { ...filtersToQueryData(filters, searchType) }
-
+  if (!isSearchTypeSupported(searchType)) {
+    return { q: searchTerm.trim() }
+  }
```

Comment:
What are the circumstances in which this would be the case? I'm not fully remembering how the supported/non-support search type thing works or why the code needs to handle both cases when only one is actually used (supported).

Reply:
If you turn on `additional_search_types` feature flag and look into the content switcher in the search results page header, the links to the types that are "Coming soon" will have the `q=cat` (or whatever) parameter set instead of just linking to `/search/video` (which will probably just redirect to the homepage since we don't have a generic media type page without a search term).
<img width="327" alt="Screenshot 2023-10-23 at 1 29 44 PM" src="https://github.com/WordPress/openverse/assets/15233243/38e0b72d-cbb2-4f99-a7f2-f05ff9d16927">